packages/cozy-stack-client/src/__tests__/CozyStackClient.spec.js

```diff
@@ -310,6 +310,26 @@ describe('CozyStackClient', () => {
       }
     })
 
+    it('should try to refresh the current token when received an invalid token error', async () => {
+      const client = new CozyStackClient(FAKE_INIT_OPTIONS)
+      global.fetch
+        .mockRejectOnce(
+          new Error(JSON.stringify({ error: 'Invalid JWT token' }))
+        )
+        .once([FAKE_APP_HTML, { status: 200 }])
+        .once([JSON.stringify({ res: 'ok' }), { status: 200 }])
+      await client.fetchJSON('GET', '/test')
+      const token = client.getAccessToken()
+      expect(token).toEqual(FAKE_APP_TOKEN)
+      const calls = global.fetch.mock.calls
+      const lastCall = calls[calls.length - 1]
+      const options = lastCall[1]
+      expect(options).toHaveProperty(
+        'headers.Authorization',
+        `Bearer ${FAKE_APP_TOKEN}`
+      )
```

Comment:
Nit: toHaveBeenLastCalledWith can also be used, trouble is you'd have to put expect.objectContaining / expect.any(object) for other arguments.

Reply:
:+1: done